src/v/pandaproxy/api/api-doc/schema_registry_header.json

```diff
@@ -2,11 +2,11 @@
   "swagger": "2.0",
   "info": {
     "title": "Pandaproxy Schema Registry",
-    "version": "1.0.2"
+    "version": "1.0.3"
```

Comment:
curious if the version bump is related to the PR of it it's just a fun bonus commit?

Reply:
Yeah, not entirely clear to me. Seems like recent changes to `schema_registry.json` also include a point version bump, but I also defer to Ben 😇

Reply:
It depends how I feel or if I remember. I think it makes sense to bump the version when a new endpoint is added.